comp/metadata/inventoryagent/inventoryagent.go

```diff
@@ -158,6 +158,7 @@ func (ia *inventoryagent) initData() {
 	ia.data["feature_fips_enabled"] = ia.conf.GetBool("fips.enabled")
 	ia.data["feature_logs_enabled"] = ia.conf.GetBool("logs_enabled")
 	ia.data["feature_cspm_enabled"] = ia.conf.GetBool("compliance_config.enabled")
+	ia.data["feature_cspm_host_benchmarks_enabled"] = ia.conf.GetBool("compliance_config.host_benchmarks.enabled")
```

Comment:
@lebauce Can you also update the [README](https://github.com/DataDog/datadog-agent/tree/main/comp/metadata/inventoryagent#format)?

Reply:
There's also tests missing for the added functionality.